<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectInfo</class>
 <widget class="QWidget" name="ObjectInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgd(55,55,55);
font: 11pt &quot;Segoe UI&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QLabel" name="scriptsLabel">
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(90,90,90);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="rotationLabel">
     <property name="text">
      <string>Rotations</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="y_2">
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="scaleLineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="xCoordLineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="scale">
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="currentScripts">
     <property name="text">
      <string>Current scripts: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="x_2">
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="x">
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="xRotation">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="z">
     <property name="text">
      <string>z:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="y">
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="yCoordLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="zCoordLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="z_2">
     <property name="text">
      <string>z:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSlider" name="zRotation">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="coordinates">
     <property name="text">
      <string>Coordinates:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="yRotation">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="addScriptButton">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="accessories-calculator"/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
